addons/website_event/i18n/website_event.pot

```diff
@@ -217,6 +217,12 @@ msgstr ""
 msgid "<span>Online Events</span>"
 msgstr ""
 
+#. module: website_event
+#: model_terms:ir.ui.view,arch_db:website_event.registration_template
+msgid ""
+"<span>Registration failed! These tickets are not available anymore.</span>"
```

Comment:
@Clement-Cardot 
Don't forget to update the translation here as well :slightly_smiling_face: (no span)

Reply:
Good catch ^^'